src/main/java/ladder/domain/Game.java

```diff
@@ -0,0 +1,76 @@
+package ladder.domain;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.IntStream;
+
+public class Game {
+    private final People people;
+    private final Ladder ladder;
+    private final Prizes prizes;
+    private final Map<String, String> result;
+
+    public Game(People people, Ladder ladder, Prizes prizes) {
+        this.people = people;
+        this.ladder = ladder;
+        this.prizes = prizes;
+        this.result = new HashMap<>();
+    }
+
+    public Map<String, String> run() {
+        List<Integer> pairs = calculateStartToEnd();
+
+        for (int i = 0; i < pairs.size(); i++) {
+            String name = people.getName(i);
+            String prize = prizes.getName(pairs.get(i));
+            result.put(name, prize);
+        }
```

Comment:
잘 수정해보셨어요~
추가로 요구사항대로 최대한 작게 만드려면 이부분도 수정되겠네요
```suggestion
        createResult(pairs);
        
            private void createResult(List<Integer> pairs) {
        for (int i = 0; i < pairs.size(); i++) {
            String name = people.getName(i);
            String prize = prizes.getName(pairs.get(i));
            result.put(name, prize);
        }
    }
```

Reply:
네, 해당 사항도 분리하여 개인 리포지토리에 반영해두었습니다!

`run` 함수 내부의 `Result` 를 만드는 `for` 문을 함수로 분리하여 한 메서드가 하나의 기능을 수행하게 하며 그러한 기능들을 조합적으로 상요하는 곳에서 가독성을 높일 수 있는 이점을 얻은 것 같습니다!

2단계까지 리뷰를 잘 해주셔서 정말 감사드립니다!